config/initializers/secure_headers.rb

```diff
@@ -13,9 +13,14 @@
     :enforce     => true,
     :default_src => 'self',
     :frame_src   => 'self',
-    :connect_src => 'self ws: wss:',
+    :connect_src => %w(self ws: wss:),
     :style_src   => 'inline self',
-    :script_src  => 'eval inline self',
-    :img_src     => ['self', '*.gravatar.com']
+    :script_src  => %w(eval inline self),
+    :img_src     => %w(self *.gravatar.com)
   }
+  if Rails.env.development? #allow webpack dev server provided assets
+    dev_server = "http://#{::Rails.configuration.webpack.dev_server.host.call}:#{::Rails.configuration.webpack.dev_server.port}"
```

Comment:
Docker & Vagrant are a fair case, it should still be able to be customized by passing env vars to the Procfile or this - anyway let's leave it at 0.0.0.0 then for now and change when we need it